src/js/view/controls/settings-menu.js

```diff
@@ -164,6 +164,28 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
         }
     }, settingsMenu);
 
+    // Visual Quality
+    model.on('change:visualQuality', (changedModel, quality) => {
+        const qualitySubMenu = settingsMenu.getSubmenu('quality');
+        const items = qualitySubMenu.getItems();
+        const item = items[0].element().querySelector('.jw-auto-label');
+        const levels = model.get('levels');
+        const { mode, reason, level } = quality;
+
+        if (mode === 'auto' || reason === 'auto') {
+            item.innerHTML = ` ${levels[level.index].label}`;
```

Comment:
Please use `textContent` when setting setting text in an element that does not contain markup.